client/ng/components/accounts/login/login.component.js

```diff
@@ -0,0 +1,58 @@
+// jshint esversion: 6
```

Comment:
Znowu problem ze wcięciami. Mają być dwie spacje. W edytorze/IDE ustawić żeby automatycznie dopasowywało wcięcia 2 spacjowe. 
Przed commitem każdy wrzucany plik powinien zostać potraktowany jakiś reformaterem żeby uniknąć tego rodzaju błędów. W Atomie to byłby dodatek `atom-beautify`, w WebStormie jest coś wbudowanego.